Makefile

```diff
@@ -18,8 +18,8 @@ parse.o: parse.c pegen.h v38tokenizer.h
 pegen.o: pegen.c pegen.h
 	$(CC) $(CFLAGS) -c $(INCLUDES) pegen.c
 
-parse.c: $(GRAMMAR) pegen.py
-	$(PYTHON) pegen.py -q -c $(GRAMMAR) -o parse.c
+parse.c: $(GRAMMAR) pegen
```

Comment:
Does that work? Shouldn't the dependency be something like `pegen/*.py`?

Reply:
Yeah, this file was not added correctly to the commit, apologies. I will update the Makefile to also move all the generated files and compilation to `pegen/`